src/option_model.py

```diff
@@ -46,3 +49,14 @@ def get_next_state(self, state: State, option: _Option) -> State:
             option,
             max_num_steps=CFG.max_num_steps_option_rollout)
         return traj.states[-1]
+
+
+class _BehaviorOptionModel(_OptionModel):
+    """An oracle option model for the BEHAVIOR env."""
+
+    def get_next_state(self, state: State,
+                       option: _Option) -> State:  # pragma: no cover
+        env = get_cached_env_instance("behavior")
+        assert option.memory["model_controller"] is not None
```

Comment:
change to `option.memory.get["model_controller"]` so that if the key doesn't exist, we'll hit this assert